proto/zitadel/admin.proto

```diff
@@ -2929,6 +2952,27 @@ message UpdateOIDCSettingsResponse {
     zitadel.v1.ObjectDetails details = 1;
 }
 
+// This is an empty request
+message GetSecurityPolicyRequest{}
+
+message GetSecurityPolicyResponse{
+    zitadel.settings.v1.SecurityPolicy policy = 1;
+}
+
+message SetSecurityPolicyRequest{
+    message Disabled{}
+    oneof setting {
+        option (validate.required) = true;
+
+        Disabled disabled = 1;
+        zitadel.settings.v1.Origins enabled = 2;
```

Comment:
to make it more clear we could say enabled_origins? or do you think its enough only enabled?